tests/unit/mock/config/parser/fortinet_fortios/fortios_full_received.py

```diff
@@ -0,0 +1,785 @@
+from netutils.config.parser import ConfigLine
+
+data = [
+    ConfigLine(config_line="config system global", parents=()),
+    ConfigLine(config_line="    set admin-concurrent enable", parents=("config system global",)),
+    ConfigLine(config_line="    set admin-console-timeout 0", parents=("config system global",)),
+    ConfigLine(config_line="    set admin-hsts-max-age 15552000", parents=("config system global",)),
+    ConfigLine(config_line="    set admin-https-pki-required disable", parents=("config system global",)),
+    ConfigLine(config_line="    set admin-https-redirect enable", parents=("config system global",)),
+    ConfigLine(
+        config_line="    set admin-https-ssl-versions tlsv1-1 tlsv1-2 tlsv1-3", parents=("config system global",)
+    ),
+    ConfigLine(config_line="    set admin-lockout-duration 60", parents=("config system global",)),
+    ConfigLine(config_line="    set admin-lockout-threshold 3", parents=("config system global",)),
+    ConfigLine(config_line="    set admin-login-max 100", parents=("config system global",)),
+    ConfigLine(config_line="end", parents=()),
```

Comment:
Is this correct? Should `end` keep context of a parent of `config system global` here?

Reply:
https://docs.fortinet.com/document/fortigate/6.0.0/cli-reference/252305/command-syntax

Reply:
Will read the guide closer.  I had thought it should be its own 'parent' but I suppose it is closing out a specific parent.  I work to get that fixed.  Also will add more tests.